hack/run-smoke.sh

```diff
@@ -31,6 +31,7 @@ function usage(){
   printf "\n--feature {FEATURE_NAME}\n\tRun a specific feature file."
   printf "\n--local ${BOOLEAN}\n\tSpecify whether you run test in local"
   printf "\n-c or --concurrent ${NUMBER}\n\Set the number of concurrent tests. Default is 1."
+  printf "\n-v or --version \n\tSet the image version. Default to current operator version"
```

Comment:
please change to `build_image_version`

Reply:
this is not yet resolved

Reply:
gosh, I changed! should be replaced again during my rebases...